src/storage/cuckoo/cuckoo.h

```diff
@@ -18,16 +18,18 @@
 #define CUCKOO_POLICY CUCKOO_POLICY_RANDOM
 #define CUCKOO_MAX_TTL (30 * 24 * 60 * 60) /* 30 days */
 #define CUCKOO_DATAPOOL NULL
+#define CUCKOO_PREFAULT false
 
-/*          name                type                default             description */
-#define CUCKOO_OPTION(ACTION)                                                                    \
-    ACTION( cuckoo_displace,    OPTION_TYPE_UINT,   CUCKOO_DISPLACE,    "# displaces allowed"   )\
-    ACTION( cuckoo_item_cas,    OPTION_TYPE_BOOL,   CUCKOO_ITEM_CAS,    "support cas in items"  )\
-    ACTION( cuckoo_item_size,   OPTION_TYPE_UINT,   CUCKOO_ITEM_SIZE,   "item size (inclusive)" )\
-    ACTION( cuckoo_nitem,       OPTION_TYPE_UINT,   CUCKOO_NITEM,       "# items allocated"     )\
-    ACTION( cuckoo_policy,      OPTION_TYPE_UINT,   CUCKOO_POLICY,      "evict policy"          )\
-    ACTION( cuckoo_max_ttl,     OPTION_TYPE_UINT,   CUCKOO_MAX_TTL,     "max ttl in seconds"    )\
-    ACTION( cuckoo_datapool,    OPTION_TYPE_STR,    CUCKOO_DATAPOOL,    "path to data pool"     )
+/*          name                      type                default             description */
+#define CUCKOO_OPTION(ACTION)                                                                          \
+    ACTION( cuckoo_displace,          OPTION_TYPE_UINT,   CUCKOO_DISPLACE,    "# displaces allowed"   )\
+    ACTION( cuckoo_item_cas,          OPTION_TYPE_BOOL,   CUCKOO_ITEM_CAS,    "support cas in items"  )\
+    ACTION( cuckoo_item_size,         OPTION_TYPE_UINT,   CUCKOO_ITEM_SIZE,   "item size (inclusive)" )\
+    ACTION( cuckoo_nitem,             OPTION_TYPE_UINT,   CUCKOO_NITEM,       "# items allocated"     )\
+    ACTION( cuckoo_policy,            OPTION_TYPE_UINT,   CUCKOO_POLICY,      "evict policy"          )\
+    ACTION( cuckoo_max_ttl,           OPTION_TYPE_UINT,   CUCKOO_MAX_TTL,     "max ttl in seconds"    )\
+    ACTION( cuckoo_datapool,          OPTION_TYPE_STR,    CUCKOO_DATAPOOL,    "path to data pool"     )\
+    ACTION( cuckoo_prefault_datapool, OPTION_TYPE_BOOL,   CUCKOO_PREFAULT,    "prefault data pool"    )
```

Comment:
`cuckoo_datapool_prefault`
same for slab.

Reply:
Done.